Added Network Console to experimental features (#809)
MicrosoftDocs/edge-developer: microsoft-edge/devtools-guide-chromium/experimental-features.md

| [Enable new CSS grid debugging features](#enable-new-css-grid-debugging-features) | 85 or later |  
| [Enable support to move tabs between panels](#enable-support-to-move-tabs-between-panels) | 85 or later |  
| [Enable webhint](#enable-webhint) | 85 or later | 
| [Enable Network Console](#enable-network-console) | 85 or later |

### Enable custom keyboard shortcuts settings tab

Provides a new **Shortcuts** page in [DevTools Settings][DevToolsCustomizeSettings] that enables matching [keyboard shortcuts][DevToolsShortcuts] in the DevTools to [VS Code][VisualstudioCode].  

> Once you have enabled the experiment, open [DevTools Settings][DevToolsCustomizeSettings] again by select `Shift`+`?`.  Navigate to the new **Shortcuts** page.  Select **DevTools (Default)** in the **Match shortcuts from preset** dropdown and select **Visual Studio Code**.  The keyboard shortcuts in the DevTools now match the shortcuts for equivalent actions in VS Code.  

Reviewer: ```suggestion
Once you have enabled the experiment, open [DevTools Settings][DevToolsCustomizeSettings] again using select `Shift`+`?`.  Navigate to the new **Shortcuts** page.  Select **DevTools (Default)** in the **Match shortcuts from preset** dropdown and select **Visual Studio Code**.  The keyboard shortcuts in the DevTools now match the shortcuts for equivalent actions in VS Code.  
```

### Enable webhint  

[webhint][WebhintMain] is an open-source tool that provides real-time feedback on the accessibility, cross-browser compatibility, security, performance, PWAs, and other common web development issues of websites.  The [webhint][WebhintMain] experiment brings the webhint feedback into DevTools in the [Issues][DevtoolsIssues] panel.  You may select the issue to see documentation on how to fix the issue and a list of the affected resources on your website.  Select a resource link to open the relevant **Network**, **Sources**, or **Elements** pane in DevTools.  

:::image type="complex" source="./media/experiments-webhint.png" alt-text="webhint feedback in the Issues panel" lightbox="./media/experiments-webhint.png":::
   webhint feedback in the Issues panel  
:::image-end:::      

<!--Available in Microsoft Edge version 85 and later.  -->  

### Enable Network Console

> **Network Console** is the working title for an experimental component for Chromium-based DevTools for making synthetic network requests over HTTP.  You may use the **Network Console** experiment during the dev-test loop for web APIs.  The **Network Console** experiment is not feature-complete during the initial release.  Early availability during the dev-test loop enables you to receive rapid iterations, experiment in the open, and see your ideas for the **Network Console** implemented quickly.  

Reviewer: ```suggestion
**Network Console** is the working title of an experiment to make synthetic network requests over HTTP.  You may use the **Network Console** experiment to call web APIs.  
```

Author: #sign-off

> Once you enable the experiment, restart your DevTools.  Open the **Network** pane, hover on the network request you are interested in changing and sending again, open the contextual menu \(right-click\), and choose **Edit and Replay**.  The **Network Console** opens and your network request information is pre-loaded for you to then edit and send.  

Reviewer: ```suggestion
After enabling the experiment, ensure you restart the DevTools. To use the Network Console:
1.  Open the **Network** pane.
1.  Find the network request that you want to change and resend.
1.  Open the contextual menu \(right-click\), and choose **Edit and Replay**. 
1.  When the **Network Console** opens, edit the network request information.
1.  Select **Send**.  
```
